main.js

```diff
@@ -57,8 +58,24 @@ const tableBuilder = (data,fields) => {
         column.appendChild(columnText)
         itemList.appendChild(column) 
     })
-    
-    itemList.appendChild(deleteButton());
+    updateTotalTime(data.time)
+    itemList.appendChild(getDeleteButton());
     return itemList;    
 }
 
+const builderCounter = (totalHours) => {
+    const totalTimeDiv = document.querySelector("#counter");
+    const contentTotalTime = `<p>${totalHours} horas de exercícios físicos </p>`;
+    totalTimeDiv.innerHTML = contentTotalTime;
+}
+
+const updateTotalTime =(time) => {
+    time = parseInt(time);
+    if(isNaN(time) || time == '' || time == null){
```

Comment:
Voce pode resumir assim:

```suggestion
    if(isNaN(time) || !time){
```
o ! vai transformar o valor para um valor booleano e depois negar ele (se for true vai pra false se for false vai pra true) 

Ex:
![image](https://user-images.githubusercontent.com/15300776/120108838-3f0e4e00-c13d-11eb-810e-2289a9b6083d.png)